packages/universal-xml-plugin/lib/attr-map.js

```diff
@@ -1,4 +1,17 @@
-// uses the same format as NODE_MAP in node-map.js
+/**
+ * Uses the same format as NODE_MAP in node-map.js
+ *
+ * @typedef {Object} AttributeMap
+ * @property {Attribute} x
+ * @property {Attribute} y
+ * @property {Attribute} width
+ * @property {Attribute} height
+ * @property {Attribute} enabled
+ * @property {Attribute} axId
+ * @property {Attribute} text
+ * @property {Attribute} visible
+ * @property {Attribute} value
+ */
```

Comment:
a better way to do this is wrap the object `{..}` in `/** @type {const} */({..})`, then you won't need to define all the properties nor will `Attribute` be necessary. same for below